models/marts/dag/fct_multiple_sources_joined.sql

```diff
@@ -6,12 +6,17 @@ with direct_source_relationships as (
     from {{ ref('int_all_dag_relationships') }}
     where distance = 1
     and parent_resource_type = 'source'
+    order by 1, 2
 ),
 
 multiple_sources_joined as (
     select
         child,
-        {{ dbt.listagg(measure='parent', delimiter_text="', '", order_by_clause='order by parent') }} as source_parents
+        {{ dbt.listagg(
+            measure='parent', 
+            delimiter_text="', '", 
+            order_by_clause='order by parent' if target.type not in ['databricks','duckdb','spark']) 
```

Comment:
do we need this at all if we're ordering in the previous CTE?

Reply:
Yes, because not all DW follow the order by of the previous data/CTE

Reply:
oh weird -- so even though the `order by` is in the CTE, the result of the listagg may not respect that ordering?